package-res/reportviewer/paginationTemplate.html

```diff
@@ -238,6 +240,21 @@
     );
   }
 
+  function toLocaleStringSupportsLocales() {
+  var number = 0;
+  try {
+    number.toLocaleString('i');
+  } catch (e) {
+    return e.name === 'RangeError';
+  }
+  return false;
```

Comment:
Should that be a "true"? Your "getLocalizedValue" function implies as much here.